haskell.html.markdown

```diff
@@ -67,10 +67,18 @@ not False -- True
 ----------------------------------------------------
 
 -- Every element in a list must have the same type.
--- Two lists that are the same
+-- These two lists are the same
 [1, 2, 3, 4, 5]
 [1..5]
 
+-- Ranges are versatile.
+['A'..'F'] -- "ABCDEF"
```

Comment:
It was mentioned, but somewhat easy to miss. I added a second example of a string as a list.